tests/ValueParsers/YearMonthDayTimeParserTest.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace ValueParsers\Test;
+
+use DataValues\TimeValue;
+use ValueParsers\EraParser;
+use ValueParsers\YearMonthDayTimeParser;
+
+/**
+ * @covers ValueParsers\YearMonthDayTimeParser
+ *
+ * @group ValueParsers
+ * @group WikibaseLib
+ * @group Wikibase
+ * @group TimeParsers
+ *
+ * @licence GNU GPL v2+
+ * @author Thiemo Mättig
+ */
+class YearMonthDayTimeParserTest extends StringValueParserTest {
+
+	/**
+	 * @deprecated since 0.3, just use getInstance.
```

Comment:
Yes, still needed for compatibility because the relevant https://github.com/DataValues/Common version is not yet released.